fmriprep/workflows/epi.py

```diff
@@ -138,6 +138,17 @@ def ref_epi_t1_registration(reportlet_suffix, inv_ds_suffix, name='ref_epi_t1_re
 
     explicit_mask_epi = pe.Node(fsl.ApplyMask(), name="explicit_mask_epi")
 
+    if settings['freesurfer']:
+        bbregister = pe.Node(
+            BBRegisterRPT(
+                contrast_type='t2',
+                init='header',
+                registered_file=True,
+                out_fsl_file=True,
```

Comment:
Reportlets work fine (mainly because we hook them up to an output sink). It's the outputs like the registered file (which I wanted to visually compare to the FLIRT-registered file) that get dropped if there isn't something else grabbing them.

Unfortunately, `needed_outputs` is an attribute of `Node`, not `BaseInterface`, so there's no simple way for an interface to mark the outputs that it uses as do-not-erase.

And yeah, the DEBUG logging is how I realized they were at least being created.